### PR TITLE
Update RELEASENOTES.adoc

### PR DESCRIPTION
**Pull request by Romeh**

### CODE REVIEW

File: RELEASENOTES.adoc [hunk 1]
```diff
@@ -277,3 +277,22 @@ NOTE: Breaking changes:
 * Issue #701: Allow custom Prometheus Histogram buckets
 * Issue #756: Spring Boot module must work without a Spring actuator
 * PR #722: Health indicator overall status can be controlled with allowHealthIndicatorToFail property
+
+== Version 1.3.0
+* Issue #822: Fixed the case where resilience4j rateLimiter seems to ignore configuration
+* Issue #816: Fixed Reactor RetryOperator throws exception wrapped by RetryExceptionWrapper
+* Issue #806: Added Provide aggregated retry.calls metric
+* Issue #803: Enabled Ciricuit breaker to emit events on disabled state
```
Comment (RobWin) [inline RELEASENOTES.adoc]: We added a new METRICS_ONLY state

File: RELEASENOTES.adoc [hunk 2]
```diff
@@ -277,3 +277,22 @@ NOTE: Breaking changes:
...
+* Issue #799: Fixed WritableStackTraceEnabled config setting in ThreadPool bulkhead configuration
+* Issue #775: Added Getter for RetryOnRetryEvent
+* Issue #769: Fixed Duplicated auto complete support in IDEA
+* Issue #765: Added support to use Retry with Feign client
+* Issue #751: Vavr upgrade to 0.10.2 
+* Issue #718: Fixed blocking behavior in Reactor Retry Operator 
+* Issue #711: Added support to use SpEL in resilienec4j spring annoations
+* Issue #751: Fixed Resilience4j Spring boot modules MetricsAutoConfiguration
+* Issue #565: Added ThreadLocal passing support to the Threadpool bulkhead implementation 
+* Issue #430: Added TimeLimkiter spring boot starter support
```
[the remark this answers is not in the document]
Comment (Romeh) [reply RELEASENOTES.adoc]: done

File: RELEASENOTES.adoc [hunk 3]
```diff
@@ -277,3 +277,22 @@ NOTE: Breaking changes:
 * Issue #701: Allow custom Prometheus Histogram buckets
 * Issue #756: Spring Boot module must work without a Spring actuator
 * PR #722: Health indicator overall status can be controlled with allowHealthIndicatorToFail property
+
+== Version 1.3.0
+* Issue #822: Fixed the case where resilience4j rateLimiter seems to ignore configuration
+* Issue #816: Fixed Reactor RetryOperator throws exception wrapped by RetryExceptionWrapper
+* Issue #806: Added Provide aggregated retry.calls metric
```
[the remark this answers is not in the document]
Comment (Romeh) [reply RELEASENOTES.adoc]: Updated

File: RELEASENOTES.adoc (hunk 1 above)
Comment (Romeh) [reply RELEASENOTES.adoc]: Updated

**Comment by Romeh** [comment]
> About #831, we have to add backward-incompatible feature in RELEASENOTES.

Good point , i have added the PR reference with the API change details

**Comment by Romeh** [comment]
@RobWin @dlsrb6342  I guess we need to mentioned that now we support Config spring customziers , right ?

**Comment by RobWin** [comment]
Yes, we have to update the documentation at several sections

**Pull request merged**